airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ads_insights_breakdowns.json

```diff
@@ -105,6 +105,10 @@
       "description": "The ID of the URL linked to by the ad.",
       "type": ["null", "string"]
     },
+    "mmm": {
```

Comment:
nit: Can we alphabetize all the new fields into the existing schema. That will make things easier to check and go through in the future